src/examples/persistence/main.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"time"
+	"unsafe"
+
+	"runtime/persistence"
+)
+
+func main() {
+	p := persistence.NewRAM(uint32(unsafe.Sizeof(byte(0)) * 100))
```

Comment:
You can assume `unsafe.Sizeof(byte)` is always 1. Doing something different would break the Go language specification: https://golang.org/ref/spec#Size_and_alignment_guarantees

I have absolutely no plans of supporting architectures that do not support byte addressing. There is hardware that doesn't support byte addressing, but it's all very special purpose (DSPs etc.)

Reply:
Yeah - you're right, that example is excessively complex - should just be 'i need a region of X bytes'